blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -57,18 +60,18 @@
     "\n",
     "As mentioned above, `aesara` provides a representation of the model logp graph in form of various `aesara` `Ops` (operators) which represent the computations to be be performed. For example `exp(x + y)` would be an `Add` `Op` with two input arguments `x` and `y`. The result of the `Add` `Op` is then inputted into an `exp` `Op`.\n",
     "\n",
-    "This computation graph doesn't say anything about how we actually *execute* this graph. Before, we would transpile this graph to C-code which would then get compiled, loaded into Python as a C-extension, and then executed. But now, we can just transpile this graph to `JAX` instead.\n",
+    "This computation graph doesn't say anything about how we actually *execute* this graph, however. Before, we would transpile this graph to C-code which would then get compiled, loaded into Python as a C-extension, and then executed. But now, we can just transpile this graph to `JAX` instead.\n",
     "\n",
-    "While this by itself is already pretty exciting, because `JAX` is capable of a whole bunch of low-level optimizations which lead to faster logp evaluations, we still have our samplers written in Python, so there is still some call-overhead. \n",
+    "While this by itself is already pretty exciting, because `JAX` is capable of a whole bunch of low-level optimizations which lead to faster model evaluation, our samplers are still written in Python, so there is still some call-overhead. \n",
```

Comment:
As I recall, JAX isn't exactly "capable of ... low-level optimizations"; it's mostly just an interface to MLIR and/or XLA, and _those_ are the providers of some low-level optimizations.